### PR TITLE
Clarify SDK ReadableSpan's parent Context (fix accidentally requiring full parent chain)

### PR DESCRIPTION
**Pull request by Oberon00**
Resolves https://github.com/open-telemetry/oteps/pull/182#discussion_r971857784

## Changes

Clarify required parent information in ReadableSpan. Technically a relaxation, but previously it was easy to overlook certain properties were required.

Related OTEP: open-telemetry/oteps#182

### CODE REVIEW

File: specification/trace/sdk.md [hunk 1]
```diff
@@ -120,21 +120,30 @@ Thus, the SDK specification defines sets of possible requirements for
...
+  A function receiving this as argument MUST must be able to reliably determine
+  whether the Span has ended
   (some languages might implement this by having an end timestamp of `null`,
   others might have an explicit `hasEnded` boolean).
 
   Counts for attributes, events and links dropped due to collection limits MUST be
   available for exporters to report as described in the [exporters](../common/mapping-to-non-otlp.md#dropped-attributes-count)
   specification.
 
+  As an exception to the authoritative set of span properties defined in the API spec,
```
Comment (Oberon00) [inline specification/trace/sdk.md:143]: This is the only (intended) content change. Even here, the intention is just to align the spec to real implementations, I don't know any that actually followed the previous spec of adding the full parent Context (which would include the full parent span if in-process, and recursively a reference to the grandparent Context, etc.)

I hope that all SDKs expose at least the full parent SpanContext but I didn't check them all (it's theoretically possible some might expose only the parent span ID, losing the parent context's trace state, trace flags, and IsRemote property).

**Comment by Oberon00** [comment]
Please tell me if I should create a tracking issue for this

**Comment by Oberon00** [comment]
I think this would be a rather important clarification to align the spec with reality.

**Comment by Oberon00** [comment]
It might be a boring PR but I still think it should not be ignored

**Review by carlosalberto — APPROVED** [review]

**Review by jmacd — APPROVED** [review]

**Comment by Oberon00** [comment]
Can this be merged?

**Pull request merged**